.github/workflows/release-module-npm.yml

```diff
@@ -5,31 +5,37 @@ on:
       env_vars:
         required: false
         type: string
-        default: '{}'
+        default: "{}"
+      node_version:
+        required: false
+        type: string
+        default: 22.x
       registry_url:
         required: false
         type: string
-        default: 'https://registry.npmjs.org'
+        default: "https://registry.npmjs.org"
       registry_scope:
         required: false
         type: string
-        default: '@digicatapult'
-      npm_build:
+        default: "@digicatapult"
+      npm_build_command:
         required: false
-        type: boolean
-        default: false
+        type: string
+        default: ""
```

Comment:
Can you see where the existing ones are used.

I'd rather we kept the existing command and created this separately.

Reply:
I've made the change in the only two places we use this flow:

https://github.com/digicatapult/tsoa-oauth-express/pull/106
https://github.com/digicatapult/sqnc-process-management/pull/352

Reply:
The current structure is extremely limiting. I don't know why we'd have this inconsistency between flows